src/webgl/framebuffer.js

```diff
@@ -187,15 +186,6 @@ export default class Framebuffer extends Resource {
         this._attachTexture({attachment, texture: object, layer: 0, level: 0});
       }
 
-      if (setFilteringParameters && object instanceof Texture2D) {
```

Comment:
I can't find any such filtering restrictions on the texture attachment, also I tested lesson-16 and picking on several deck-gl layers, all looking good.